### ISSUE TITLE
bug: Capacitor AdMob doesn't display correctly (not a plugin issue)

### ISSUE THREAD

**Issue by alokjkumar** [ionic-team/capacitor#2887]
# Bug Report

## Capacitor Version
`npx cap doctor` output:
```
   Capacitor Doctor

Latest Dependencies:

  @capacitor/cli: 2.1.0

  @capacitor/core: 2.1.0

  @capacitor/android: 2.1.0

  @capacitor/electron: 2.1.0

  @capacitor/ios: 2.1.0

Installed Dependencies:

  @capacitor/electron not installed


  @capacitor/core 2.0.0

  @capacitor/cli 2.0.0

  @capacitor/ios 2.0.0

  @capacitor/android 2.0.1

[success] Android looking great! 👌
[error] Xcode is not installed
```

## Affected Platform(s)

- [x] Android
- [ ] iOS
- [ ] Electron
- [ ] Web

## Current Behavior
When showing banner ads using this plugin: https://github.com/rdlabo-team/capacitor-admob the banner ad does not show correctly. Instead half of the screen is occupied with a splashscreen as shown in the screenshot.

![96302079_240696787244836_2266361938957041664_n](https://user-images.githubusercontent.com/1263680/81426843-ad0e5e80-9151-11ea-91d2-89d486eb9409.jpg)

## Expected Behavior
The ad should show at the bottom of the page content with no splashscreen beneath it.

## Sample Code or Sample Application Repo
app.component.ts:
```
const { AdMob } = Plugins;

initializeApp() {
  this.platform.ready().then(() => {
    if(!this.platform.is('mobileweb') && !this.platform.is('desktop')) {
      StatusBar.setStyle({
         style: this.platform.is('ios') ? StatusBarStyle.Light : StatusBarStyle.Dark
      });
      StatusBar.setBackgroundColor({
         color: '#EC008C'
      });
      SplashScreen.hide();
      AdMob.initialize();
    }
  });
}
```
home.page.ts:
```
  async showBanner() {
    // Tried triggering with OnInit, AfterViewInit and Click event, none work
    const result = await AdMob.showBanner(this.bannerOptions)
      .catch(e => console.log(e));
    if (result === undefined) {
      return;
    }
  }
```

I am not able to share the entire repo because of an NDA but I'm confident this is a native issue not a JavaScript one...

## Reproduction Steps
In CLI:
ionic build && npx cap sync
npx cap open android

In Android Studio:
Run -> Debug 'app'
(debugged on USB attached Android device)

## Other Technical Details

`npm --version` output: 6.14.4

`node --version` output: v10.16.3

`@ionic/angular` version: ^5.0.7

`@angular/common` version: ^9.1.0

`@rdlabo/capacitor-admob` version: ^1.0.1

## Other Information
Debug console output on event trigger:
```
D/ViewRootImpl@81677c8[MainActivity]: ViewPostIme pointer 0
D/ViewRootImpl@81677c8[MainActivity]: ViewPostIme pointer 1
V/Capacitor/Plugin: To native (Capacitor plugin): callbackId: 95859117, pluginId: AdMob, methodName: showBanner
V/Capacitor: callback: 95859117, pluginId: AdMob, methodName: showBanner, methodData: {"adId":"ca-app-pub-3940256099942544\/2934735716","adSize":"SMART_BANNER","position":"BOTTOM_CENTER"}
D/Capacitor/Plugin/AdMob: Ad ID: ca-app-pub-3940256099942544/2934735716
D/InputMethodManager: HSIFW - flag : 0 Pid : 31491
D/DynamitePackage: Instantiating com.google.android.gms.ads.ChimeraAdManagerCreatorImpl
I/Ads: Use RequestConfiguration.Builder().setTestDeviceIds(Arrays.asList("C0B5FAD20881037744FF202AC0CE54A2") to get test ads on this device.
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->getInt(Ljava/lang/Object;J)I (greylist, linking, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->getObject(Ljava/lang/Object;J)Ljava/lang/Object; (greylist, linking, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->putObject(Ljava/lang/Object;JLjava/lang/Object;)V (greylist, linking, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->putInt(Ljava/lang/Object;JI)V (greylist, linking, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->getLong(Ljava/lang/Object;J)J (greylist,core-platform-api, linking, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->putLong(Ljava/lang/Object;JJ)V (greylist, linking, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->allocateInstance(Ljava/lang/Class;)Ljava/lang/Object; (greylist, linking, allowed)
W/com.blynd.dev: Accessing hidden method Llibcore/io/Memory;->peekLong(JZ)J (greylist, reflection, allowed)
    Accessing hidden method Llibcore/io/Memory;->pokeLong(JJZ)V (greylist, reflection, allowed)
    Accessing hidden method Llibcore/io/Memory;->pokeInt(JIZ)V (greylist, reflection, allowed)
    Accessing hidden method Llibcore/io/Memory;->peekInt(JZ)I (greylist, reflection, allowed)
    Accessing hidden method Llibcore/io/Memory;->pokeByte(JB)V (greylist, reflection, allowed)
    Accessing hidden method Llibcore/io/Memory;->peekByte(J)B (greylist, reflection, allowed)
    Accessing hidden method Llibcore/io/Memory;->pokeByteArray(J[BII)V (greylist, reflection, allowed)
    Accessing hidden method Llibcore/io/Memory;->peekByteArray(J[BII)V (greylist, reflection, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->arrayBaseOffset(Ljava/lang/Class;)I (greylist,core-platform-api, linking, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->arrayIndexScale(Ljava/lang/Class;)I (greylist, linking, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->getLong(Ljava/lang/Object;J)J (greylist,core-platform-api, reflection, allowed)
    Accessing hidden field Ljava/nio/Buffer;->address:J (greylist, reflection, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->getInt(Ljava/lang/Object;J)I (greylist, reflection, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->putInt(Ljava/lang/Object;JI)V (greylist, reflection, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->getLong(Ljava/lang/Object;J)J (greylist,core-platform-api, reflection, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->putLong(Ljava/lang/Object;JJ)V (greylist, reflection, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->getObject(Ljava/lang/Object;J)Ljava/lang/Object; (greylist, reflection, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->putObject(Ljava/lang/Object;JLjava/lang/Object;)V (greylist, reflection, allowed)
I/System.out: (HTTPLog)-Static: isSBSettingEnabled false
    (HTTPLog)-Static: isSBSettingEnabled false
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->compareAndSwapObject(Ljava/lang/Object;JLjava/lang/Object;Ljava/lang/Object;)Z (greylist, linking, allowed)
I/DynamiteModule: Considering local module com.google.android.gms.ads.dynamite:0 and remote module com.google.android.gms.ads.dynamite:21400
I/DynamiteModule: Selected remote version of com.google.android.gms.ads.dynamite, version >= 21400
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->getObject(Ljava/lang/Object;J)Ljava/lang/Object; (greylist, linking, allowed)
V/FA: Connecting to remote service
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->getInt(Ljava/lang/Object;J)I (greylist, linking, allowed)
D/FA: Connected to remote service
V/FA: Processing queued up service tasks: 1
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->getInt(Ljava/lang/Object;J)I (greylist, linking, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->getInt(Ljava/lang/Object;J)I (greylist, linking, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->putObject(Ljava/lang/Object;JLjava/lang/Object;)V (greylist, linking, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->getInt(Ljava/lang/Object;J)I (greylist, linking, allowed)
    Accessing hidden method Lsun/misc/Unsafe;->getObject(Ljava/lang/Object;J)Ljava/lang/Object; (greylist, linking, allowed)
I/DynamiteModule: Considering local module com.google.android.gms.ads.dynamite:0 and remote module com.google.android.gms.ads.dynamite:21400
    Selected remote version of com.google.android.gms.ads.dynamite, version >= 21400
V/Capacitor/Plugin/AdMob: Notifying listeners for event onAdLoaded
D/Capacitor/Plugin/AdMob: No listeners found for event onAdLoaded
V/Capacitor/Plugin/AdMob: Notifying listeners for event onAdSize
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->getInt(Ljava/lang/Object;J)I (greylist, linking, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->getLong(Ljava/lang/Object;J)J (greylist,core-platform-api, linking, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->putInt(Ljava/lang/Object;JI)V (greylist, linking, allowed)
W/com.blynd.dev: Accessing hidden method Lsun/misc/Unsafe;->putLong(Ljava/lang/Object;JJ)V (greylist, linking, allowed)
W/com.blynd.dev: Long monitor contention with owner gads-6 (31742) at long ajt.i(int)(:com.google.android.gms.policy_ads_fdr_dynamite@21460000@21460000.297791526.297791526:0) waiters=6 in void com.google.android.ads.tasks.w.a() for 119ms
I/System.out: (HTTPLog)-Static: isSBSettingEnabled false
I/System.out: (HTTPLog)-Static: isSBSettingEnabled false
```

**Comment by jcesarmobile** [comment]
can you provide a sample app?

**Comment by alokjkumar** [comment]
I've made a repo with all api keys removed. It's unlikely to compile now but you can see the source code.

https://github.com/alokjkumar/blynd-nokeys

**Comment by jcesarmobile** [comment]
I need an app that works, if not I can't reproduce

**Comment by alokjkumar** [comment]
I've added you as a collaborator to the private repo

**Comment by jcesarmobile** [comment]
I can reproduce, but I don't think this is a Capacitor issue since the plugin is the one messing with the views.

The splash you see is not really the splash screen plugin, is the background of `NoActionBarLaunch` theme, you can remove the `<item name="android:background">@drawable/splash</item>` from `styles.xml` and the problem will go away, but instead, you'll get a grey/white screen before the splash plugin. But you can maybe configure the color to match your app theme so is less noticeable, in example `<item name="android:background">#ec008c</item>`.

TLDR: replace `<item name="android:background">@drawable/splash</item>` with `<item name="android:background">#ec008c</item>` in `styles.xml`.
